definitions/variable/emissions/carbon-removal.yaml

```diff
@@ -2,8 +2,71 @@
     description: Gross removals of carbon dioxide (CO2) from atmospheric origin
       or biomass through deliberate human activities
     unit: Mt CO2/yr
-    notes: Values should be reported as positive numbers
-- Carbon Removal|{Carbon-Removal Option}:
-    description: Gross removals of carbon dioxide (CO2) by {Carbon-Removal Option}
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+
+- Carbon Removal|Geological Storage:
+    description: Gross removals of carbon dioxide (CO2) from atmospheric origin
+      or biomass through geological storage
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+- Carbon Removal|Geological Storage|{Geogological Storage Source}:
+    description: Gross removals of carbon dioxide (CO2) through {Geogological Storage Source}
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+
+- Carbon Removal|Land Use:
+    description: Gross removals of carbon dioxide (CO2) through land-based sinks
+      (e.g., afforestation, soil carbon enhancement, biochar)
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+- Carbon Removal|Land Use|{Land Removal Option}:
+    description: Gross removals of carbon dioxide (CO2) through {Land Removal Option}
+    unit: Mt CO2/yr
+    tier: 2
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+
+- Carbon Removal|Long-Lived Materials:
+    description: Gross removals of carbon dioxide (CO2) through long-lived materials
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+    navigate: Carbon Removal|Materials
+    engage: Carbon Removal|Materials
+- Carbon Removal|Long-Lived Materials|{Long-Lived Materials}:
+    description: Gross removals of carbon dioxide (CO2) through {Long-Lived Materials}
+    unit: Mt CO2/yr
+    tier: 2
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+    navigate: Carbon Removal|Materials|{Long-Lived Materials}
+    engage: Carbon Removal|Materials|{Long-Lived Materials}
+
+- Carbon Removal|Ocean:
+    description: Gross removals of carbon dioxide (CO2) through ocean sinks
```

Comment:
Also here I would report net removals instead of gross. Especially for ocean liming, there is the possibility to do it without CCS, which would result in rather large gross emissions and removals, but relatively smaller net removals. Reporting only gross removals would be misleading. We can add gross removals later to also have this information

Reply:
I would have to disagree here (and therefore by proxy also disagree with the point above). If I am an ocean modeller and I want to understand what is the carbon removal due to OAE based on the alkalinity addition, how would I do this if we mix in the process emissions?

I think we need to report them both (removal through alkalinity addition and emissions generated through creating the alkalinity in the first place). 

There is an interesting argument *where* the positive emissions should live. Should they be reported in the "Carbon Removal" tree? Or should they be part of industry emissions (since this would be an industrial process)?

Reply:
Another argument: how do we calculate total cumulative stored carbon? If we report only net, we will underestimate what the model is actually putting underground.